src/components/Facet/index.js

```diff
@@ -3,15 +3,31 @@ import PropTypes from 'prop-types'
 import Button from '../Button'
 import { CheckBoxInput } from '../Inputs'
 import classnames from 'classnames'
+import { t } from '@lingui/macro'
 import './styles.scss'
 
 const ShowHideMore = ({id, isOpen, toggleOpen}) => {
   return (
     <Button
-      ariaLabel='Show all values'
+      ariaLabel={t({
+        comment: 'Aria label for Show|Hide More Button',
+        message: 'Show all values'
+      })}
       ariaPressed={isOpen}
       className='facet__show-hide'
-      label={isOpen ? 'show less' : 'show all'}
+      label={
+        isOpen
+        ? 
+        t({
+          comment: 'Message shown when list is opened',
+          message: 'show less'
+        })
+        :
+        t({
+          comment: 'Message shown on list is closed',
+          message: 'show all'
+        })
+      }
```

Comment:
What are you thoughts on doing conditional logic like this for which translation to show. The other option is to remove the conditional outside of the translation and use a select within the message for conditional logic. I have an example of that [here](https://github.com/RockefellerArchiveCenter/dimes/blob/16303b04d302bef83dacebe34b7635aa71785b02/src/components/MyListExportActions/index.js#L19-L24) which I also commented below.

Reply:
Hmmm...at first glance the `select` method seems a lot cleaner. Do you have a preference/thoughts from working through this?